### ISSUE TITLE
Version picker's role and state are not defined for screen readers

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#948]
### Problem

The version picker https://github.com/readthedocs/sphinx_rtd_theme/blob/5ba05cd1faa2ce22723b437d2b1d0f265dfcd225/sphinx_rtd_theme/versions.html#L4 does not offer clues in its markup to allow screen readers to determine its state, such as if it is open or closed.

### Reproducible Project

https://docs.citusdata.com/en/stable/

#### Error Logs/Results

![MAS4 1 2-Role and state of Read the docs menu is not defined](https://user-images.githubusercontent.com/48225118/85180210-76dff680-b248-11ea-8802-b4f963baca3d.jpg)

#### Expected Results

Control should be compliant with [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

https://www.youtube.com/watch?v=73EdfgAPw8E&feature=youtu.be&t=20m45s

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4
